BioMed.Api/BioMed.Infrastructure/Persistence/Repositories/RepositoryBase.cs

```diff
@@ -0,0 +1,59 @@
+﻿using BioMed.Domain.Entities;
+using BioMed.Domain.Exceptions;
+using BioMed.Domain.Interfaces.Repositories;
+using Microsoft.EntityFrameworkCore;
+
+namespace BioMed.Infrastructure.Persistence.Repositories
+{
+    public class RepositoryBase<T> : IRepositoryBase<T> where T : EntityBase
+    {
+        protected readonly BioMedDbContext _context;
+
+        public RepositoryBase(BioMedDbContext context)
+        {
+            _context = context;
+        }
```

Comment:
I think it would be better if we check the **context** for null inside the **RepositoryBase**
 constructor.

Reply:
Thanks for your recommendation